<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxEmployeeForm</class>
 <widget class="QMainWindow" name="BoxEmployeeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Box Office Eemployee</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyResources.qrc">
    <normaloff>:/Images/make movie logo transparent.png</normaloff>:/Images/make movie logo transparent.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtNewTransaction">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>New Transaction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtLogOut">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Time logged in:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTimeLoggedIn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
